fix_permit_coordinates.py

```
@@ -0,0 +1,487 @@
+        print(f"🎯 Enhancing {len(permits)} permits with coordinates...")
+        
+        enhanced_permits = []
+        geocoded_count = 0
+        
+        for permit in permits:
+            enhanced_permit = permit.copy()
+            
+            # Try to find address in various fields
+            address = None
+            for field in ['address', 'project_address', 'site_address', 'location', 'property_address']:
+                if field in permit and permit[field]:
```

Ruff (0.12.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)